src/core/annotation.js

```diff
@@ -3668,7 +3708,11 @@ class InkAnnotation extends MarkupAnnotation {
     return ink;
   }
 
-  static createNewAppearanceStream(annotation, xref) {
+  static async createNewAppearanceStream(
+    annotation,
+    xref,
+    /* params */ _unused
```

Comment:
How about simply leaving it as follows, since defining a *different* but likewise unused variable here seems unnecessary?
```suggestion
    params
```